exporter/cassandraexporter/exporter_traces.go

```diff
@@ -23,12 +23,10 @@ type tracesExporter struct {
 }
 
 func newTracesExporter(logger *zap.Logger, cfg *Config) (*tracesExporter, error) {
-	cluster := gocql.NewCluster(cfg.DSN)
-	session, err := cluster.CreateSession()
+	cluster := newCluster(cfg)
 	cluster.Keyspace = cfg.Keyspace
```

Comment:
Can this line be transferred into func `newCluster` too?

Reply:
I thought about it, but ```cluster.Keyspace ```is not in ```initializeLogKernel```